src/coreclr/tools/aot/ILCompiler/ILCompiler.csproj

```diff
@@ -79,6 +79,7 @@
 
   <ItemGroup Condition="'$(NativeAotSupported)' == 'true'">
     <CustomLinkerArg Condition="'$(CrossBuild)' == 'true' and '$(_hostArchitecture)' == '$(_targetArchitecture)' and '$(_hostOS)' != 'windows' and '$(_IsApplePlatform)' != 'true'" Include="--gcc-toolchain=$(ROOTFS_DIR)/usr" />
+    <CustomLinkerArg Condition="'$(_IsApplePlatform)' == 'true'" Include="-ld_classic" />
```

Comment:
This shouldn't be needed?

Reply:
IIRC it uses the released package, not the one we just built, so it is necessary.